libcontainer/nsenter/nsexec.c

```diff
@@ -139,6 +152,9 @@ void nsexec()
 		close(fd);
 	}
 
+	close(self_tfd);
```

Comment:
These are the fds of the directories (/proc/{self,<pid>}/ns). We open these before the loop, use them in all iterations, and finally close them after the loop ends. Closing them inside the loop would break all iterations but the first one.